src/pages/playerScreen/PlayerScreen.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useState } from 'react';
+import ReactPlayer from 'react-player';
+
+import Components from '../../components';
+import { movieService } from '../../services';
+
+const PlayerScreen = () => {
+	const [detailScreenMovieList, setdetailScreenMovieList] = useState([]);
+
+	const getdetailScreenMovieList = async () => {
+		try {
+			const response = await movieService._getMovieListByType(
+				'featured',
+				10
+			);
+			setdetailScreenMovieList(response.data?.data);
+		} catch (err) {
+			console.error(err);
+		}
+	};
+
+	useEffect(() => {
+		getdetailScreenMovieList();
+	}, []);
+
+	return (
+		<>
+        <ReactPlayer
```

Comment:
create a separate organism for player with dynamic props, it'll be handy to change anything in the player